app/views/layouts/application.html.erb

```diff
@@ -24,6 +24,10 @@
 
     <%= render partial: "shared/analytics/google" %>
 
+    <% if show_tracking_pixels? && !Rails.env.test? %>
```

Comment:
Do we want this active in develop? `show_tracking_pixels? && Rails.env.production?` would prevent us from polluting tracking data with local development.